packages/core/helper-plugin/lib/src/components/RelationInput/RelationInput.js

```diff
@@ -85,22 +89,26 @@ export const RelationInput = ({
                     </button>
                   }
                 >
-                  {href ? (
-                    <BaseLink disabled={disabled} href={href}>
-                      {title}
-                    </BaseLink>
-                  ) : (
-                    title
-                  )}
+                  <Box paddingTop={1} paddingBottom={1}>
+                    {href ? (
+                      <BaseLink disabled={disabled} href={href}>
+                        {title}
+                      </BaseLink>
+                    ) : (
+                      title
+                    )}
+                  </Box>
 
-                  <Badge
-                    borderSize={1}
-                    borderColor={`${badgeColor}200`}
-                    backgroundColor={`${badgeColor}100`}
-                    textColor={`${badgeColor}700`}
-                  >
-                    {isDraft ? 'Draft' : 'Published'}
-                  </Badge>
+                  {isDraft !== undefined && (
```

Comment:
mmh I think I would need more info on all data format possible because we have 3 cases to handle:
- D&P is activated, entry is published
- D&P is activated, entry is draft
- D&P is not activated, what the published state looks like?

if I only use `badgeColor` I have a "Published" badge appearing because isDraft is false 
wdyt?

Reply:
> D&P is not activated, what the published state looks like?

That's a good question for @maevalienard actually :)

Reply:
If the Draft & Published feature is not enabled, then we do not show any badge :)

Reply:
Ok then. @ronronscelestes I'll need to check if/ how I could pass this information to you. I think it's not ideal to derive it from each item, since this is rather a global content-type setting of the target model.

Reply:
sounds good, let me know - we can easily iterate on it / update it when we'll have the right information!

Reply:
Note: checkbox added to `feature/relations-main-view` to improve isDraft data when D&P not activated